File: 10699/10699.js
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   10699.js                                           :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2024/03/02 16:32:31 by jihoolee          #+#    #+#             */
/*   Updated: 2024/03/02 16:33:20 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

let today = new Date();

let year = today.getFullYear();
let month = String(today.getMonth() + 1).padStart(2, '0');
let day = String(today.getDate()).padStart(2, '0');

console.log(`${year}-${month}-${day}`);

File: 10699/10699.c
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   10699.c                                            :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2024/03/02 16:33:44 by jihoolee          #+#    #+#             */
/*   Updated: 2024/03/02 16:35:57 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <stdio.h>
#include <time.h>

int	main(void)
{
	time_t		t;
	struct tm	tm;
	int			year;
	int			month;
	int			day;

	t = time(NULL);
	tm = *localtime(&t);
	year = tm.tm_year + 1900;
	month = tm.tm_mon + 1;
	day = tm.tm_mday;
	printf("%04d-%02d-%02d\n", year, month, day);
	return (0);
}
